compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -329,7 +332,7 @@ bool TaggedShapeAnalyzer::init(const luci::CircleTranspose *front_transpose,
     return true;
   };
 
-  // check c3
+  // check c2
```

Comment:
(note) re-numbering condition
`c1` is removed in previous PR - https://github.com/Samsung/ONE/pull/14167#discussion_r1789782341